### DISCUSSION TITLE
Why is ImageGlass open an URL with Analytics-Parameters when no valaid Configuration is found?

### DISCUSSION THREAD

**Discussion by Snickbrack** [d2phap/ImageGlass#984] [General] [answered]
On Start-Up this URL will be opened when no valid config is found:
`Process.Start($"https://imageglass.org/docs/app-configs?utm_source=app_{App.Version}&utm_medium=app_click&utm_campaign=incompatible_configs");`

Why do you need these analytic parameters?

**Comment by d2phap** [comment, marked as answer]
Most of the links from ImageGlass have tracking code from Google Analytics, these are to help me know the number of users interacting to the app (updating new version, downloading new theme packs, having problem with configuration, ...)
